### PR TITLE
Release v12.0.0 to next tag on NPM

### PR DESCRIPTION
**Pull request by GarthDB**
## Description

Merging this PR will trigger a release to `@adobe/spectrum-tokens` `v12.0.0` with the `latest` tag on [NPM](https://www.npmjs.com/package/@adobe/spectrum-tokens?activeTab=versions).

Additionally, I've also added deprecation metadata to all of the express tokens.

## Motivation and Context

This will help us get back to Semver with a more informative version number than just using beta releases.

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,6 @@
 # Spectrum Tokens
 
----
-
-# Note: This project is in beta right now.
-
----
-
-Design tokens are all the values needed to construct and maintain a design system — spacing, color, typography, object styles, animation, etc. — represented as data. These can represent anything defined by design: a color as a RGB value, an opacity as a number, an animation ease as Bezier coordinates. They’re used in place of hard-coded values in order to ensure flexibility and unity across all product experiences.
+Design tokens are all the values needed to construct and maintain a design system — spacing, color, typography, object styles, animation, etc. — represented as data. These can represent anything defined by design: a color as an RGB value, an opacity as a number, and an animation ease function as Bezier coordinates. They're used in place of hard-coded values to ensure flexibility and unity across all product experiences.
```
Comment (misterbrownlee) [inline README.md:3]: Ideally we can change this to say something more like "Design tokens are values that *help* construct and maintain a design system."  Having this say "all of the values needed" is a bad expectation to set.

Comment (misterbrownlee) [reply README.md:3]: that said ... I'm fine fixing it as a patch after this goes out.

File: README.md [hunk 2]
```diff
@@ -15,11 +9,70 @@ Design tokens are directly integrated into our component libraries, UI kits, and
...
+
+- adding new token
+- changing a value intentionally
+- adding deprecation metadata and aliased new tokens where applicable
+
+#### Major
+
+n+1.0.0 releases are for breaking changes.
+
+- deleting tokens
```
Comment (misterbrownlee) [inline README.md:55]: should we mention "renaming that's not a typo" here?

**Review by misterbrownlee — APPROVED** [review]
I approve, and we can talk about my comments and make fixes for them later

File: README.md (hunk 2 above)
Comment (GarthDB) [reply README.md:55]: That should really be deleting deprecated tokens. We deprecate, we create the new token name, we alias the old token name to the new token name, and we eventually delete the deprecated token.

**Pull request merged**